changelog_unreleased/typescript/10085.md

```diff
@@ -0,0 +1,28 @@
+#### Heuristics for TS classes that has broken heritage clause (#10085 by @sosukesuzuki)
```

Comment:
```suggestion
#### Improve visual separation between header and body in classes with multiline headers (#10085 by @sosukesuzuki)
```

Also, this change isn't only for TS. Let's move the change log entry to the `javascript` folder.

E.g. this is pure JS:

**Prettier pr-10085**
[Playground link](https://deploy-preview-10085--prettier.netlify.app/playground/#N4Igxg9gdgLgprEAucAbAhgZ0wAgGIQQA6UOOA9OTpALY0IwllwAe8UAJrgELoBOOYExz0YACwgcAFAEpBAXzJkS8kABoQEAA4wAltEzJQ-PhADuABX4JDKdKjPoAnoY0AjPujABrODADKWl66UADmyDB8AK5wGnA0bnAcHEkAMuhhUeihcAR8NOgwemHIIOhRMBDqIGIwNKgA6mK68JhBYHD+Ni26AG4tTqVg2NUhmHB8MBaeoQXIAGb24xoAVpgs3J4+fv7o9KkhcAtLsSBrLP4hoahwAIpREPDHqMsgQXzjfKVu6Imo1Vo+CEYA1dBxxMgABwABg0gIg4wani0pUBcE+vSOGgAjg94NNtLYypgALRQOBJJLVPhwXG6GnTbJzJCLF6ncY0XQRaLsq43e6PI4sk4aGC-UHgsTIABMos8ulQVwAwhA6OhSuiAKzVKLjAAqv1srNevRiAElOAx-GAgToAIKcfwwJw3Z7jeTyIA)
```sh
--parser babel
```

**Input:**
```jsx
class Foo
  // comment
  extends Bar {
  method() {}    
}
```

**Output:**
```jsx
class Foo
  // comment
  extends Bar
{
  method() {}
}

```